pages/clubInfoPage.js

```diff
@@ -0,0 +1,149 @@
+import Head from 'next/head'
+import clientPromise from '../lib/mongodb'
+import PhotoSlider from "../components/photoslider.js";
+{/* import NavBar from "../components/NavBar"*/}
```

Comment:
Any reason this is commented out? If not, let's add it back in and add the navbar to the page b/c the actual web app will need that

Reply:
fixed